hazelcast-client/src/main/resources/hazelcast-client-failover-config-3.12.xsd

```diff
@@ -16,13 +16,15 @@
   -->
 
 <xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns="http://www.hazelcast.com/schema/client-config"
            elementFormDefault="qualified"
+           targetNamespace="http://www.hazelcast.com/schema/client-config"
            attributeFormDefault="unqualified">
 
     <xs:element name="hazelcast-client-failover">
         <xs:complexType>
             <xs:choice minOccurs="1" maxOccurs="unbounded">
-                <xs:element name="try-count" type="xs:int" minOccurs="1" maxOccurs="1"/>
+                <xs:element name="try-count" type="non-space-string" minOccurs="1" maxOccurs="1"/>
```

Comment:
So, we should not use xs:int anywhere to support replacers, is that the case?
I have just checked we have usages of it in hazelcast-client-config-3.12.xsd. Should all be removed?

Reply:
I revert this part of the schema change. As you said, this is a general problem with replacers. We have a solution for this in the spring schema: https://github.com/hazelcast/hazelcast/blob/d47c1b5962146de2711fa0e4950fa0cfa91b7083/hazelcast-spring/src/main/resources/hazelcast-spring-3.12.xsd#L3368-L3370
I believe this approach should be used by the other XSDs as well, but this needs no retrospective change.

However, I still think adding `xmlns` and `targetNamespace` needs retrospective change.